uui-core/src/data/processing/views/tree/hooks/common/usePatchTree.ts

```diff
@@ -57,4 +56,30 @@ export function usePatchTree<TItem, TId, TFilter = any>(
             ...tree.visible.getParams(),
         });
     }, [tree, patch]);
+
+    const patchFn = useCallback((updated: IMap<TId, TItem> | IImmutableMap<TId, TItem>) => {
```

Comment:
This is how it is organized. I couldn't find any easier way to apply all the changes to the tree along with estimate/start date/due date change.